### ISSUE TITLE
Regression: 'v-slot' directive must be owned by a custom element, but 'template' is not. eslint(vue/valid-v-slot)

### ISSUE THREAD

**Issue by Shinigami92** [rashfael/eslint-plugin-vue-pug#11]
![image](https://user-images.githubusercontent.com/7195563/174961538-8afcde11-f545-4787-8765-6f147ac5d926.png)

- Since v0.5.1
- I don't know why, but downgrading to v0.5.0 doesn't solve this bug, maybe it's related to a (peer)dependency?

**Comment by Grawl** [comment]
just faced this

**Comment by rashfael** [comment]
You probably need to downgrade vue-eslint-parser-template-tokenizer-pug instead. Can you give me a larger snippet?

**Comment by Shinigami92** [comment]
Confidential code send via Discord PM

**Comment by rashfael** [comment]
I boiled it down to this:
```pug
root
  nest-1
    nest-1-1
      nest-1-1-1(
      )
    nest-1-2
  nest-2
```

Wich gives me these tokens:
<details>
<summary> intermediate tokens</summary>

```json
[
  {
    "type": "StartTag",
    "name": "root",
    "rawName": "root",
    "selfClosing": false,
    "attributes": [],
    "range": [
      21,
      25
    ],
    "loc": {
      "start": {
        "line": 1,
        "column": 21
      },
      "end": {
        "line": 1,
        "column": 25
      }
    }
  },
  {
    "type": "StartTag",
    "name": "nest-1",
    "rawName": "nest-1",
    "selfClosing": false,
    "attributes": [],
    "range": [
      27,
      33
    ],
    "loc": {
      "start": {
        "line": 2,
        "column": 1
      },
      "end": {
        "line": 2,
        "column": 7
      }
    }
  },
  {
    "type": "StartTag",
    "name": "nest-1-1",
    "rawName": "nest-1-1",
    "selfClosing": false,
    "attributes": [],
    "range": [
      36,
      44
    ],
    "loc": {
      "start": {
        "line": 3,
        "column": 2
      },
      "end": {
        "line": 3,
        "column": 10
      }
    }
  },
  {
    "type": "StartTag",
    "name": "nest-1-1-1",
    "rawName": "nest-1-1-1",
    "selfClosing": false,
    "attributes": [],
    "range": [
      48,
      64
    ],
    "loc": {
      "start": {
        "line": 4,
        "column": 3
      },
      "end": {
        "line": 5,
        "column": 4
      }
    }
  },
  {
    "type": "EndTag",
    "name": "nest-1-1-1",
    "loc": {
      "start": {
        "line": 5,
        "column": 4
      },
      "end": {
        "line": 6,
        "column": 2
      }
    },
    "range": [
      64,
      67
    ]
  },
  {
    "type": "EndTag",
    "name": "nest-1-1",
    "loc": {
      "start": {
        "line": 5,
        "column": 4
      },
      "end": {
        "line": 6,
        "column": 2
      }
    },
    "range": [
      64,
      67
    ]
  },
  {
    "type": "StartTag",
    "name": "nest-1-2",
    "rawName": "nest-1-2",
    "selfClosing": false,
    "attributes": [],
    "range": [
      67,
      75
    ],
    "loc": {
      "start": {
        "line": 6,
        "column": 2
      },
      "end": {
        "line": 6,
        "column": 10
      }
    }
  },
  {
    "type": "EndTag",
    "name": "nest-1-2",
    "loc": {
      "start": {
        "line": 6,
        "column": 10
      },
      "end": {
        "line": 7,
        "column": 1
      }
    },
    "range": [
      75,
      77
    ]
  },
  {
    "type": "EndTag",
    "name": "nest-1",
    "loc": {
      "start": {
        "line": 6,
        "column": 10
      },
      "end": {
        "line": 7,
        "column": 1
      }
    },
    "range": [
      75,
      77
    ]
  },
  {
    "type": "EndTag",
    "name": "root",
    "loc": {
      "start": {
        "line": 6,
        "column": 10
      },
      "end": {
        "line": 7,
        "column": 1
      }
    },
    "range": [
      75,
      77
    ]
  },
  {
    "type": "StartTag",
    "name": "nest-2",
    "rawName": "nest-2",
    "selfClosing": false,
    "attributes": [],
    "range": [
      77,
      83
    ],
    "loc": {
      "start": {
        "line": 7,
        "column": 1
      },
      "end": {
        "line": 7,
        "column": 7
      }
    }
  },
  {
    "type": "EndTag",
    "name": "nest-2",
    "loc": {
      "start": {
        "line": 7,
        "column": 7
      },
      "end": {
        "line": 8,
        "column": 0
      }
    },
    "range": [
      83,
      84
    ]
  }
]
```
</details>

the root tag gets closed too early, which seems to be related to the multiline attribute braces.

**Comment by rashfael** [comment]
Fixed in 0.5.2.
